<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterestChargeCheckingsWizardPage</class>
 <widget class="QWizardPage" name="InterestChargeCheckingsWizardPage">
  <attribute name="title">
    <string>Interest / Charges</string>
   </attribute>
   <layout class="QVBoxLayout">
    <item>
     <widget class="QLabel" name="qLabel1" native="true">
      <property name="text" stdset="0">
       <string>If necessary, enter information about interest or charges here. KMyMoney will create transactions and cleares them directly for you.</string>
      </property>
      <property name="wordWrap">
        <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="Line2">
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="Spacer8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <spacer name="Spacer7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="textLabel1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Payee</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KMyMoneyPayeeCombo" name="m_payeeEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox1">
          <property name="title">
           <string>Interest</string>
          </property>
          <layout class="QVBoxLayout">
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="KMyMoneyCategory" name="m_interestCategoryEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="AmountEdit" name="m_interestEdit">
               <property name="resetButtonVisibility" stdset="0">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KMyMoneyDateInput" name="m_interestDateEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox2">
          <property name="title">
           <string>Charges</string>
          </property>
          <layout class="QVBoxLayout">
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="KMyMoneyCategory" name="m_chargesCategoryEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="AmountEdit" name="m_chargesEdit">
               <property name="resetButtonVisibility" stdset="0">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KMyMoneyDateInput" name="m_chargesDateEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="Spacer6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="Spacer9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyPayeeCombo</class>
   <extends>KComboBox</extends>
   <header>kmymoneypayeecombo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AmountEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">amountedit.h</header>
  </customwidget>	  
 </customwidgets>
 <tabstops>
 </tabstops>
 <buttongroups>
 </buttongroups>
</ui>
